reduce heterogeneity section by 80% (#91)
jaybee84/ml-in-rd: content/05.heterogeneity.md

### Managing disparities in data generation is required for robust rare disease analyses

Rare disease data from can suffer from artifacts introduced by batch, assay platform, specimen quality or other non-biological phenomena. 
The consequences of these artifacts are amplified in rare diseases which often have few samples and heterogeneous phenotypes.

> Furthermore, datasets are often pieced together from multiple small studies in which important biological characteristics are confounded by the aforementioned non-biological variables. 

Reviewer: ```suggestion
Furthermore, datasets are often pieced together from multiple small studies where biological characteristics are confounded by technical variables. 
```

> Active dialogue with the data generators or domain experts may result in unexpected insight into potential sources of variation. 

Reviewer: ```suggestion
Collaboration with data generators or domain experts may result in unexpected insight into potential sources of variation. 
```

Reviewer: Alternatively, remove `Active` and start with `Dialogue`.

> One example of this experienced by the authors occurred when studying the disease neurofibromatosis type 1. 

Reviewer: ```suggestion
The authors experienced this when studying neurofibromatosis type 1 (NF1). 
```

> These datasets were, unbeknownst to the computational biologists, generated from samples obtained with vastly different surgical techniques, resulting in substantial biological differences that were a consequence of process, not reality. 

Reviewer: ```suggestion
The NF1 datasets were comprised of samples obtained with different surgical techniques, resulting in biological differences that were a consequence of sample collection, rather than ...
```

Reviewer: Add citation also?

> Consequently, careful assessment of confounding factors and implementation of normalization methods is critical to identifying biologically meaningful features within a dataset. 

Reviewer: ```suggestion
Consequently, careful assessment of and accounting for confounding factors is critical to identifying biologically meaningful features within a dataset. 
```

> Assessment of confounding factors and heterogeneity in rare disease datasets is perhaps most easily performed using unsupervised learning approaches (such as clustering and dimensionality reduction). 

Reviewer: ```suggestion
Assessment of confounding factors and heterogeneity is perhaps most easily performed using unsupervised learning approaches. 
```

> K-means clustering or hierarchical clustering can be used to characterize the structure present in data such as genomic or imaging data. [@doi:10.1186/1471-2105-9-497;@doi:10.1109/JBHI.2013.2276766]. 

Reviewer: ```suggestion
K-means clustering or hierarchical clustering can be used to characterize the structure present in genomic or imaging data. [@doi:10.1186/1471-2105-9-497;@doi:10.1109/JBHI.2013.2276766]. 
```

> Similarly, a variety of dimensionality reduction methods can be used to visualize heterogeneity and confounders, including multidimensional scaling, principal components analysis, t-distributed stochastic neighbor embedding (t-SNE), and uniform manifold approximation and projection (UMAP), among others. [@doi:10.1007/978-3-540-33037-0_14; @doi:10.1098/rsta.2015.0202; @https://lvdmaaten.github.io/publications/papers/JMLR_2008.pdf; @arXiv:1802.03426]

Reviewer: ```suggestion
Similarly, dimensionality reduction methods can be used to visualize heterogeneity and confounders, including multidimensional scaling, principal components analysis, t-distributed stochastic neighbor embedding (t-SNE), and uniform manifold approximation and projection (UMAP), among others. [@doi:10.1007/978-3-540-33037-0_14; @doi:10.1098/rsta.2015.0202; @https://lvdmaaten.github.io/publications/papers/JMLR_2008.pdf; @arXiv:1802.03426]
```

I would pick 2 fewer examples -- `among others` can do some heavy lifting!

All of these methods can be used to identify batch effects and other structure in the data, though some (like t-SNE and UMAP) require parameters that can affect the output and it's interpretation [@doi:10.23915/distill.00002;@arXiv:1802.03426]. 

> Therefore, obtaining a clear interpretation from these methods requires understanding the underlying approach as well as parameter sweeping.  

Reviewer: ```suggestion
Therefore, obtaining a clear interpretation from these methods requires understanding the underlying approach and parameters.  
```
